hack/verify-govet.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+
+GO_VERSION=($(go version))
+
+if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.4') ]]; then
```

Comment:
We can go ahead and merge this, but in a follow up, make this tolerate 1.4* so we'll still check for 1.4.1, etc